devops/build_sdks.py

```diff
@@ -219,13 +219,13 @@ def main():
         build_python(args)
     if build_all or 'java' in langs_to_build:
         build_java(args)
-        build_java_docs(args)
```

Comment:
shouldn't we also `build_java_docs` if `--language=java`? since docs will stay in sync with source changes

Reply:
Negative. See the `sdk` repo for examples. The docs build based upon anything being changed (thereby guaranteeing it's right). The build action for the language is for the code itself. Documentation is cross-language concern.

Reply:
I see - this is what had me confused.  so we'd want docs build as a separate workflow (which will be fleshed out later as you commented).  and it should use `build_all`.  makes sense to me!

Reply:
that build-docs workflow will want to set up all these deps (e.g. Node, Dotnet) that I'd added but you're 100% right that that should be separate.